desktop/apps/auctions2/templates/index.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { withLayout } from 'desktop/components/react/main_layout'
+
+function Index (props) {
+  const {
+    Layout: { Header, Body, Footer },
+    Templates: { Meta }
+  } = props
+
+  return (
+    <div>
+      <Header>
+        <Meta />
+      </Header>
+      <Body>
+        hello!
+      </Body>
+      <Footer />
+    </div>
```

Comment:
I want to reiterate that the point of all this is to clear a path for migrating forward, and nothing more. I want to make things _forward compatible_ and then, once that's in place, start thinking about backwards compatibility (as @craigspaeth mentioned in the first PR) which will then quite possibly fit into the work being done over in reaction-force. The two systems are largely incompatible; this makes them compatible (up until a point), but the main blocker was the layout structure for force in general.